pandas/tests/io/json/test_ujson.py

```diff
@@ -559,6 +560,14 @@ def test_encode_long_conversion(self):
         assert output == json.dumps(long_input)
         assert long_input == ujson.decode(output)
 
+    def test_dumps_ints_larger_than_maxsize(self):
+        # GH34395
+        big_num = sys.maxsize + 1
+        encoding = ujson.dumps(big_num)
+
+        assert encoding == json.dumps(big_num)
+        assert ujson.loads(encoding) == big_num
```

Comment:
This would require fixing both to_json and read_json which could be a rather large PR. I think OK to just compare the output of to_json to the expected string for now